libs/remix-simulator/test/blocks.ts

```diff
@@ -1,12 +1,12 @@
 /* global describe, before, it */
-const Web3 = require('web3')
-const RemixSim = require('../index.js')
+import Web3 from 'web3'
+import { Provider } from '../src/index'
 const web3 = new Web3()
```

Comment:
are you creating a new instance of Web3 on every file o.O ?

Reply:
yes